src/module.js

```diff
@@ -117,7 +117,7 @@ _.extend(Marionette.Module.prototype, Backbone.Events, {
       Marionette,
       Backbone.$, _,
       customArgs
-    ]);
+    ], true);
```

Comment:
> kestutis-stalmokas  wrote: test for `lodash _.name === 'lodash'`

totally don't do the `.name` check as that will be borked on minified code.

> megawac: It's tricky because in lodash its `flatten(collection, deep)` and in underscore its `flatten(collection, shallow)`

Yap you can do `_.flatten(collection, !!_.flattenDeep)`. In Underscore it will be `false` so `_.flatten` will be deep and in lodash it will be `true` so also a deep flatten.